### PR TITLE
refactor(settings): Add missing menu items and adjust items order

### PR DESCRIPTION
**Pull request by kvviecien**
- Add "Email Aliases" under "Mail" (link to https://runbox.com/mail/account_alias)
- Add "Sub-Accounts" under "Account" - visible only to main accounts (link to https://runbox.com/mail/account_account)
- Split "Account & Security" into "Account" and "Security"
- Match icons in the side menu with the ones in the main settings screen

### CODE REVIEW

File: src/app/account-app/account-app.component.ts [hunk 1]
```diff
@@ -32,17 +33,27 @@ export class AccountAppComponent {
...
+                rmmapi:      RunboxWebmailAPI,
+                router:      Router,
     ) {
         this.sideMenuOpened = !mobileQuery.matches;
         this.mobileQuery.changed.subscribe(mobile => {
             this.sideMenuOpened = !mobile;
         });
 
+        rmmapi.me.subscribe((me: RunboxMe) => {
+            if (me.owner) {
```
Comment (tadzik) [inline src/app/account-app/account-app.component.ts:50]: Minor: could be shortened to `this.isMainAccount = !me.owner`

File: src/app/account-app/account-app.component.html [hunk 2]
```diff
@@ -106,43 +113,45 @@
...
                     </mat-expansion-panel-header>
                     <mat-list-item routerLink="/account/subscriptions">
                         <p mat-line>
-                            <mat-icon mat-list-icon svgIcon="autorenew"></mat-icon> Your Subscriptions
+                            <mat-icon mat-list-icon svgIcon="folder-star-multiple"></mat-icon> Your Subscriptions
                         </p>
                     </mat-list-item>
                     <mat-list-item routerLink="/account/plans">
                         <p mat-line>
-                            <mat-icon mat-list-icon svgIcon="hot-tub"></mat-icon> Plans & Upgrades
```
Comment (tadzik) [inline src/app/account-app/account-app.component.html:140]: Rip `hot-tub` :(

**Review by tadzik — APPROVED** [review]
Looks good to me :)

File: src/app/account-app/account-app.component.html (hunk 2 above)
Comment (kvviecien) [reply src/app/account-app/account-app.component.html:140]: Never forget [*]

File: src/app/account-app/account-app.component.ts (hunk 1 above)
Comment (kvviecien) [reply src/app/account-app/account-app.component.ts]: Did the thing

**Pull request merged**